### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by greggmiller**
Updated 27th March, copy to include Luke Sleeman in acknowledgements and WH medical staff who have contributed.

☝️ Provide a short & descriptive title for this PR.

Provide a more detailed summary of the changes in this PR here.  What does it do?

[ ] `release-notes.txt` has been updated.
(👆 delete this line if your changes have no user facing features)

## Changes

- Dot points explaining what is changed
- Did the thing
- Changed some stuff

## Screenshots

If your PR changes anything visual please paste in some screenshots of the app here.  Make sure you are working off the designs in zeplin.  If your PR makes some significant visual tweaks feel free to include one of the designers as a reviewer, so they can check off and comment on what is changing.

## Things to note

Include anything else the other Devs or Designers on the project should know?  Will there be work you are deferring now, which done in later follow up PRs?   Does this change anything about the code base they need to be aware of?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ If you've never used Github here's a [Github101.md](docs/GITHUB101.md) to get yo
 
 Everything is co-ordinated through github issues.  The current development process is:
 
-👨‍⚕️👩‍⚕️ **Greg Miller** and other Western Health staff provide features, content and suggestions via GitHub issues.  Greg & Co can create new issues for features to add, things that need changing, etc.  
+👨‍⚕️👩‍⚕️ **Gregg Miller** and other Western Health staff provide features, content and suggestions via GitHub issues.  Greg & Co can create new issues for features to add, things that need changing, etc.
```
Comment (lukesleeman) [inline README.md]: Doh!  I don't belive there was a typo in your name!

File: README.md [hunk 2]
```diff
@@ -167,6 +167,11 @@ Currently the project is being primarily co-ordinated via the GDG Melbourne slac
 * Brett Morgan - Flutter dev
 * Eric Jiang - Flutter dev, and QA
 * Dylan Lange - Flutter dev
+* Jonathan Galtieri - Anaesthetist
```
Comment (lukesleeman) [inline README.md:170]: Great work 👏

**Review by lukesleeman — APPROVED** [review]
Looks good!  @greggmiller you can click the merge button now if you want :-)

File: README.md (hunk 1 above)
Comment (kimngyn) [reply README.md]:
```suggestion
👨‍⚕️👩‍⚕️ **Gregg Miller** and other Western Health staff provide features, content and suggestions via GitHub issues.  Gregg & Co can create new issues for features to add, things that need changing, etc.  
```

**Review by kimngyn — DISMISSED** [review]

**Comment by greggmiller** [comment]
Good point @kimngyn

**Review by suesitran — APPROVED** [review]
LGTM

**Pull request merged**